java/arcs/core/host/BUILD

```diff
@@ -1,13 +1,39 @@
-load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_kt_library")
+load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_kt_library", "arcs_kt_schema")
 
 licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
+API_SRCS = [
+    "HandleHolder.kt",
+    "Particle.kt",
+]
+
+arcs_kt_schema(
+    name = "schemas",
+    srcs = [
+        "ArcHostContext.arcs",
+    ],
+    package = "arcs.core.host",
+)
+
+arcs_kt_library(
+    name = "api",
```

Comment:
Could you move these sources into a new `api` package? (as in, subfolder, with its own blaze target)